client/packages/programs/src/JsonForms/components/EncounterProgramEvent.tsx

```diff
@@ -31,6 +31,8 @@ const Options = z
     at: z.enum(['before', 'start']).optional(),
     eventType: z.string(),
     displayType: z.enum(['string', 'number']).optional(),
+    multiline: z.boolean().optional(),
```

Comment:
Added this since having ARV code + description makes it too long to fit in the text field even once all component widths have been fixed...